core/app.go

```diff
@@ -95,6 +96,8 @@ func NewApp(configFlag string) (*App, error) {
 	if err = cfg.InitLogging(); err != nil {
 		return nil, err
 	}
+	configJSON, _ := json.Marshal(*cfg)
```

Comment:
Also we're skipping over the error here. Are we sure there's no case in which the JSON serialization can fail?